packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/field/AdditionalPropertiesSchemaField.tsx

```diff
@@ -16,50 +16,50 @@ const ISO_TIMESTAMP_REGEX = /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{3}Z/
 
 // Types that correspond to the different input fields that the annotation editor supports
 export enum PropertyType {
```

Comment:
Just so I understand, why was this simple enum not converted into a static array?

Reply:
No real reason, it's basically equivalent if the keys and values are the same. But that invariant could easily be broken unintentionally, so I think I'll also change this to a string union defined by array